code/__HELPERS/unsorted.dm

```diff
@@ -1343,3 +1343,6 @@ B --><-- A
 		c_dist++
 
 	return L
+
+/proc/subtypesof(typepath)

```

Comment:
No really tho there's no reason I can see to make this a proc instead of a macro
Make it macro for that sweet sweet inline speed